controllers/fqdnnetworkpolicy_controller.go

```diff
@@ -45,6 +45,7 @@ type FQDNNetworkPolicyReconciler struct {
 var (
 	ownerAnnotation        = "fqdnnetworkpolicies.networking.gke.io/owned-by"
 	deletePolicyAnnotation = "fqdnnetworkpolicies.networking.gke.io/delete-policy"
+	AaaaLookupsAnnotation  = "fqdnnetworkpolicies.networking.gke.io/aaaa-lookups"
```

Comment:
`aaaaLookupsAnnotation` to align with the case of the other variables

Reply:
applied